### DISCUSSION TITLE
(Example) Where can I find cloud-optimized data?

### DISCUSSION THREAD

**Discussion by abarciauskas-bgse** [cloudnativegeo/cloud-optimized-geospatial-formats-guide#41] [Q&A]
What are the best places to search for cloud-optimized data? Specifically, I'm looking for publicly available geoparquet and flatgeobuf datasets.

**Comment by PostholerCom** [comment]
Make your own with GDAL:

`ogr2ogr dest.fgb source.shp`

**Reply by wildintellect** [reply]
That doesn't address the request of wanting to have publicly accessible web based resources for use in examples or testing. @abarciauskas-bgse I would start with https://source.coop/

**Comment by PMassicotte** [comment]
I converted GEBCO 2023 (https://www.gebco.net/data_and_products/gridded_bathymetry_data/#global) bathymetric data to COG: 

```
/vsicurl/https://gebco2023.s3.valeria.science/gebco_2023_land_cog.tif
/vsicurl/https://gebco2023.s3.valeria.science/gebco_2023_sub_ice_topo_cog.tif
```
